src/dom_types/values.rs

```diff
@@ -33,6 +33,35 @@ impl From<&CSSValue> for CSSValue {
     }
 }
 
+// ----------- ToCssValue impls ------------
+
+// impl<T: ToString> ToCssValue for T
+#[doc(hidden)]
+pub trait ToCssValueForToString {
+    fn to_css_value(&self) -> CSSValue;
+}
+
+impl<T: ToString> ToCssValueForToString for T {
```

Comment:
I think it's basically the same like `impl<T: ToString> From<T> for CSSValue {`. => Try to delete `impl<T: ToString> From<T> for CSSValue {` and run `cargo make verify`.

Reply:
I left this so style.add function can handle values that impl ToString, so if someone want to add styles using this function they found it handy too.